packages/scrubber-bar/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This presents a new complexity - how do we trickle down the .eslint config to all child packages from top-level of repo? https://github.com/internetarchive/iaux/blob/master/.eslintrc.json

Reply:
I was able to extend the base iaux `.eslintrc.json` file with only minor fixes to the style so now this one adheres to the top-level one.